src/screentovnc.h

```diff
@@ -114,6 +113,7 @@ class ScreenToVnc : public QObject
     static void unixTermSignalHandler(int unused);
 
     static Recorder *m_recorder;
+    static Orientation m_orientation;
```

Comment:
Just a note: Usually we have instance members marked with `m_` and static members marked with `s_` (and then there may be `g_` for global) but this (old) piece of code apparently doesn't follow those rules. No need to correct it now as the other static member is like similarly named, I just wanted to mention that for the future, if you come across that elsewhere.

That said, this is not the only weird coding convention thing in this code base.

Reply:
heh. yea, i assumed the convention was `m_` for member (static or instance) and `s_` for global static for some reason. the actual convention is.....better, i guess :D